test_pyganja.py

```diff
@@ -38,10 +38,50 @@ def test_draw_points(self):
         gs = GanjaScene()
         gs.add_objects([random_euc_mv().value[0:8] for i in range(10)], static=False)
         with open('test_file.html', 'w') as test_file:
-            print(generate_full_html(str(gs), sig=layout.sig, grid=True, scale=1.0, gl=False), file=test_file)
+            test_file.write(generate_full_html(str(gs), sig=layout.sig, grid=True, scale=1.0, gl=True))
         render_cef_script(str(gs), sig=layout.sig, grid=True, scale=1.0, gl=False)
 
 
+class TestPGA3DDrawing(unittest.TestCase):
+
+    def setUp(self):
+        from clifford.pga import layout, blades
+        import numpy as np
+        self.layout = layout
+        self.blades = blades
+        self.np = np
+
+    def random_point(self):
+        return sum(c * b for c, b in
+                   zip(self.np.random.randn(3), [self.blades["e023"], self.blades["e013"], self.blades["e012"]])) + \
+               self.blades["e123"]
```

Comment:
nit: could make it a bit shorter:
```suggestion
        return self.blades["e123"] + sum(
            c * self.blades[b] for c, b in zip(self.np.random.randn(3), ["e023", "e013", "e012"]
        )
```